src/app/zeko/sequencer/lib/zeko_sequencer.ml

```diff
@@ -823,6 +852,30 @@ let%test_unit "apply commands and commit" =
     ; name = "gql-uri"
     }
   in
+  let da_websocket = "ws://localhost:8546" in
+  let da_private_key =
+    "0x35f9400884bdd60fdd1a769ebf39fa1c5b148072e68a5b2c8bc9ac2227c192b2"
+  in
```

Comment:
.

Reply:
it's the same testing key as here https://github.com/zeko-labs/zeko/blob/main-berkeley/src/app/zeko/da-layer/hardhat.config.ts#L5

Reply:
Make the websocket information configurable.